tests/test_qubit_device.py

```diff
@@ -343,7 +346,7 @@ def test_results_created(self, mock_qubit_device_extract_stats, monkeypatch, mea
 
         qscript = QuantumScript(measurements=[measurement])
 
-        with monkeypatch.context() as m:
+        with monkeypatch.context():
```

Comment:
is this needed to make the mock device work? same question in general for when the context variable is not used (and now removed) throughout the PR

Reply:
...good question :sweat_smile:

Reply:
The answer is no :) I think this was just copy-paste legacy, and the mocking is actually happening in the fixtures, so no need to open the context in the test function itself.